elasticsearch/src/main/java/org/hibernate/search/elasticsearch/impl/ToElasticsearch.java

```diff
@@ -418,6 +424,32 @@ private static JsonObject convertRemoteMatchQuery(RemoteMatchQuery query) {
 		return wrapQueryForNestedIfRequired( query.getField(), matchQuery );
 	}
 
+	private static JsonObject convertRemoteSimpleQueryStringQuery(RemoteSimpleQueryStringQuery query) {
+		JsonArray fieldArray = new JsonArray();
+		for ( Field field : query.getFields() ) {
+			StringBuilder sb = new StringBuilder( field.getName() );
+			if ( field.getBoost() != DEFAULT_BOOST ) {
+				sb.append( BOOST_OPERATOR ).append( field.getBoost() );
+			}
+			fieldArray.add( sb.toString() );
+		}
+
+		JsonBuilder.Object queryBuilder = JsonBuilder.object()
+				.addProperty( "query", query.getQuery() )
+				.addProperty( "defaultOperator", query.isMatchAll() ? "and" : "or" );
```

Comment:
It's `default_operator`, not `defaultOperator`. It may work in ES2, but I've seen cases where ES5 dropped such undocumented syntax, so we should stay away from such syntax.